### DISCUSSION TITLE
[Google Auth] Account choice

### DISCUSSION THREAD

**Discussion by kapik** [capawesome-team/capacitor-firebase#311] [Q&A] [answered]
Hi ! 👋 

I use the plugin in an Ionic application.
Everything works fine.
On the mobile version (in Chrome directly), the `signInWithGoogle` method offers me all the available accounts.
But on the application version, the plugin uses the main account and does not propose the list.

Is it possible to have the choice of the account to use?
Thanks for your help 🚀

**Comment by robingenz** [comment, marked as answer]
Yes, you should be able to select a account. Do you log out correctly? (See https://github.com/robingenz/capacitor-firebase-authentication/issues/129)

**Reply by kapik** [reply]
Oh cool! 

Thanks for the help.
I'm logging out fine, but just like you need to log in with the web layer using `signInWithCredential`, you also need to log out with it (I'm still using AngularFire on the side, for now).

Maybe add in the readme here https://github.com/capawesome-team/capacitor-firebase/blob/main/packages/authentication/docs/firebase-js-sdk.md ?

Thanks again

**Reply by robingenz** [reply]
Good idea! I added a usage example: https://github.com/capawesome-team/capacitor-firebase/commit/11be69777d4f282831d1334aef542575d4f3366e
